gimme

```diff
@@ -413,57 +417,58 @@ fi
 
 while [[ $# -gt 0 ]]; do
 	case "${1}" in
-		-h|--help|help|wat)
+		-h | --help | help | wat)
 			_old_ifs="$IFS"
 			IFS=';'
 			awk '/^#\+  / {
 				sub(/^#\+  /, "", $0) ;
 				sub(/-$/, "", $0) ;
 				print $0
-			}' "$0" | while read line ; do
-				eval "echo \"$line\""
-			done
+			}' \
+				"$0" | while read -r line; do
+					eval "echo \"$line\""
+				done
 			IFS="$_old_ifs"
 			exit 0
-		;;
-		-V|--version|version)
+			;;
+		-V | --version | version)
 			echo "${GIMME_VERSION}"
 			exit 0
-		;;
-		-l|--list|list)
+			;;
+		-l | --list | list)
 			_list_versions
 			exit 0
-		;;
-		-f|--force|force)
+			;;
+		-f | --force | force)
 			force=1
-		;;
+			;;
 		*)
 			break
-		;;
+			;;
 	esac
 	shift
 done
 
-if [[ -n "${1}" ]] ; then
+if [[ -n "${1}" ]]; then
 	GIMME_GO_VERSION="${1}"
 fi
-if [[ -n "${2}" ]] ; then
+if [[ -n "${2}" ]]; then
 	GIMME_VERSION_PREFIX="${2}"
 fi
 
 case "${GIMME_ARCH}" in
 	x86_64) GIMME_ARCH=amd64 ;;
 	x86) GIMME_ARCH=386 ;;
 	arm64)
-		if [[ "${GIMME_GO_VERSION}" < "1.5" ]]; then
+		if [[ "${GIMME_GO_VERSION}" ]] <"1.5"; then
```

Comment:
For clarity: `[` is an alias for `test`, and with the I/O redirection removed the effective argv for the cmdline to be invoked was then `[ "${GIMME_GO_VERSION}" ]`; `test` without a flag, only a string, is a test for the string being not-null, so just is "true as long as $GIMME_GO_VERSION is defined with a non-empty value".  The falsehood comes from the shell being able to set up input redirection into the `test` command.  The scenario reduced to `true < /nonexistent`